ZenPacks/zenoss/Microsoft/Windows/WinService.py

```diff
@@ -88,39 +85,34 @@ def getMonitored(self, datasource):
         return rtn
 
     def monitored(self):
-        """Return True if this service should be monitored. False otherwise."""
+        return self.is_monitored()
 
+    def is_monitored(self):
```

Comment:
move the guts of is_monitored() into monitored() and remove the is_monitored() function